packages/shared/src/lib/boot.ts

```diff
@@ -78,5 +85,20 @@ export async function getBootData(app: string, url?: string): Promise<Boot> {
     credentials: 'include',
     headers: { app, 'Content-Type': 'application/json' },
   });
-  return res.json();
+  const result = await res.json();
+
+  try {
+    const features = await decrypt(
+      result.exp.f,
+      process.env.NEXT_PUBLIC_EXPERIMENTATION_KEY,
+      'AES-CBC',
+      128,
+    );
+
+    result.exp.features = JSON.parse(features);
+  } catch (error) {
+    // TODO features-loading-optimization log error
```

Comment:
@rebelchris can I maybe track decrypt error event here? We didn't track it in previous implementation? You mentioned there is a track error event somewhere already?

Reply:
Yes we can add it :)
You can look at `useError` for a global one.
Or something like `RegistrationInitializationError` 

Probably the later is better so we can have a unique key basically.

Reply:
Since we didn't catch previously we will leave as is, we can add catch in WT-1882